vocab/credentials/v2/issuer-dependent.html

```diff
@@ -66,15 +68,58 @@
+            Applications using additional, user-dependent vocabularies of their own are encouraged to 
+            create their own formal vocabularies, and corresponding JSON-LD context definitions, 
+            to ensure the interoperability of the credentials they produce and/or consume.
         </p>
         <p>
-            There may be cases, however, when the creation of such formal vocabularies and context definitions is too much of a burden, such as for closed applications, experiments, or examples. The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> has been constructed in such a way that issuers may use new, hitherto undefined terms without further ado, and these terms will be mapped onto URLs of the form <code>https://www.w3.org/ns/credentials/issuer-dependent#TERM</code>.
+            Not all users desire to define JSON-LD vocabularies or to make use of them with W3C Verifiable Credentials. 
+            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> has been constructed 
+            in such a way that issuers may use new, as-yet undefined terms without additional effort, 
+            and these terms will be mapped onto URLs of the form <code>https://www.w3.org/ns/credentials/issuer-dependent#TERM</code>.
         </p>
         <p>
             This document provides a proper landing place for those URLs, without any additional definitions for the terms themselves. 
         </p>
+        <section id="dynamic-term" style="color: #FF5C5C;"></section>
     </section>
-
+    <script>
+      function htmlEncode(str){
+          return String(str).replace(/[^\w. ]/gi, function(c){
+              return '&#'+c.charCodeAt(0)+';';
+          });
+      }
+      (async ()=>{
+        // handle issuer / hodler defined terms in fragment
+        if (window.location.hash.length){
+          const [_, term] = window.location.hash.split('#')
+          let el = document.getElementById('dynamic-term')
+          el.id = term;
+          const encodedTerm = htmlEncode(term);
+          el.innerHTML = `    
+<h2>${encodedTerm}</h2>
+<p>
+The term <code><strong>${encodedTerm}</strong></code> (<code>https://www.w3.org/ns/credentials/issuer-dependent#${encodedTerm}</code>)
+was not defined in a JSON-LD context.
+</p>
+<p>
+Please consider contacting the <a href="https://www.w3.org/2018/credentials/#issuer">issuer</a> 
+or <a href="https://www.w3.org/2018/credentials/#holder">holder</a> if you believe this is an error.
+</p>
+<p>You may also consider reviewing:</p>
+<ul>
+  <li><a href="https://www.w3.org/TR/json-ld11/#terms">www.w3.org/TR/json-ld11/#terms</a></li>
+  <li><a href="https://www.w3.org/2018/credentials">www.w3.org/2018/credentials</a></li>
+  <li><a href="https://github.com/w3c/vc-specs-dir">w3c/vc-specs-dir</a></li>
+</ul>
+<p>You might also consider searching for the term in a search engine, possibly with "+jsonld" in the query.</p>
```

Comment:
```suggestion
```

It's not clear what would result from that suggestion either now, or over time. Don't think the suggestion is useful beyond the advice given above.